### PR TITLE
Support PostgreSQL 10 and 11, and use Python 3

### PR DESCRIPTION
**Pull request by varjoranta**
Fixed the issue with packaging and psycopg2 compilation. Had to use the pre-compiled binary package with `no-manylinux1` flag. Works now with Postgres 10 and 11 as well. Moved away from Python 2 and Upstart.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@ --with python-virtualenv
+	dh $@ --with python-virtualenv --python python3 --preinstall=no-manylinux1
```
Comment (andreasscherman) [inline debian/rules]: I think we need `--with systemd` here as well, as we're using <10 compat level. We could also bump the compat, but that might bring other side-effects with it.  😄

File: debian/control [hunk 2]
```diff
@@ -3,14 +3,14 @@ Section: non-free/net
 Priority: extra
 Maintainer: Hannu Varjoranta <hannu.varjoranta@spotify.com>
 Build-Depends:
-  python (>= 2.7),
-  libpq-dev (<<9.7),
+  python3 (>= 3.5),
+  libpq-dev,
   debhelper (>= 9),
   dh-virtualenv (>= 1.0),
```
Comment (andreasscherman) [inline debian/control:9]: I think we need to include `dh-systemd (>= 1.5) ` here as well, given the repo has a compat <10

**Review by andreasscherman — COMMENTED** [review]
Looks good! Just some small comments on systemd.

Have you ran it through something alike `2to3` or equivalent to find the parts of the code which breaks for Python 3? Might be some subtle cases it might catch!

**Comment by varjoranta** [comment]
I used `2to3` and it is good to go from that perspective. The extra stuff it wanted to do is ok to ignore, i.e. turning iterators to lists when not needed, and the raw_input/input part was already handled properly. Good to go if you 👍

**Comment by andreasscherman** [comment]
LGTM! 👍

**Pull request merged**